tests/utils/utils.go

```diff
@@ -501,21 +509,102 @@ func SendNativeTokenSource(
 		nativeTokenSource,
 		input.DestinationBlockchainID,
 		input.DestinationBridgeAddress,
-		bridgedAmount,
+		amount,
 	)
 	teleporterUtils.ExpectBigEqual(event.Amount, scaledAmount)
 
 	return receipt, event.Amount
 }
 
+func DepositAndApproveWavaxForFees(
```

Comment:
Whats the difference between `DepositAndApproveWavaxForFees` and `DepositAndApproveNativeForFees`? I think we can likely combine them into a single helper if we add an ABI binding for the `IWrappedNativeToken` interface, which `ExampleWAVAX` and `NativeTokenDestination` both implement. 

Naming nit: `WAVAX` should be all caps in the middle of a name if we end up using it still.

Reply:
I factored them out, but the event structs are specific to each contract, so I've removed those.